site_settings/models.py

```diff
@@ -23,6 +23,22 @@ class EmergencyHours(BaseSiteSetting):
                 FieldPanel("enable"),
                 FieldPanel("link_text"),
             ],
-            heading="Emergency Hours"
+            heading="Emergency Hours",
         )
     ]
+
+
+@register_setting(icon="mail")
+class ContactInfo(BaseSiteSetting):
+    report_a_problem = models.URLField(
+        max_length=200,
+        null=False,
+        blank=False,
+        default=(
+            "https://www.lib.uchicago.edu/" "research/help/ask-librarian/ask-contact/"
+        ),
```

Comment:
I'm not familiar with this kind of string concatenation (though I see it works in the interpreter). I'm not sure if this is related to some of the errors I'm seeing with `report_a_problem` but I'm thinking it would be better to use the standard `+` operator or an f-string or something else, however, I'm also not sure that we need concatenation here. Can't it just be one string?

I'm also not sure that `default` is doing what you want it to do here. It's only used when new model instances are created and a value isn’t provided. It would put the default value if a user saved with a blank field, assuming it wasn't required. I feel like you might have a different intent.